src/app/plan/duck/operations.ts

```diff
@@ -375,6 +285,74 @@ function* fetchPlansGenerator() {
     return { e, isSuccessful: false };
   }
 }
+const getPVs = (dispatch, updatedPlansPollingResponse, newObjectRes) => {
+  const matchingPlan = updatedPlansPollingResponse.updatedPlans
+    .filter(p => p.MigPlan.metadata.name === newObjectRes.data.metadata.name)
+    .pop();
+
+  const pvSearchStatus = matchingPlan ? getPlanPVs(matchingPlan) : null;
+  if (pvSearchStatus.success) {
+    dispatch(Creators.updatePlan(matchingPlan.MigPlan));
+    dispatch(pvFetchSuccess());
+    dispatch(commonOperations.alertSuccessTimeout('Found PVs!'));
```

Comment:
We have an issue to ditch this notification, could knock out another one with this PR: https://github.com/fusor/mig-ui/issues/318